simplq/src/components/pages/PageNotFound.jsx

```diff
@@ -1,21 +1,13 @@
 import React from 'react';
-import { Typography, Button, Grid } from '@material-ui/core';
+import Button from '../common/Button';
+import styles from '../../styles/pageNotFound.module.scss';
 
 function PageNotFound(props) {
   return (
-    <>
-      <Typography align="center" gutterBottom>
-        Something went wrong. TODO: Report issue github link
-      </Typography>
-      {/* Link to github page for raising issue. */}
-      <Grid container style={{ marginTop: '30px' }} justify="center">
-        <Grid item>
-          <Button variant="contained" color="primary" onClick={() => props.history.push('/')}>
-            Go home
-          </Button>
-        </Grid>
-      </Grid>
-    </>
+    <div className={styles.main}>
+      <h1>404: Page not found</h1>
+      <Button onClick={() => props.history.push('/')}>Go home</Button>
+    </div>
```

Comment:
I'll raise a low priority jira to style this page...

Reply:
#193

Reply:
I checked out you branch on local... and it's looking pretty good... Can we also give a button:

"Report a Bug"

that redirects to https://github.com/SimplQ/simplQ-frontend/issues/new ?

Reply:
Yeah, I'll add this